validator/src/main/resources/validations/healthcheck-metric-validation.yml

```diff
@@ -0,0 +1,3 @@
+-
+  validationType: "ecs-describe-task"
+  callingType: "none"
```

Comment:
Ensure all files end with a newline.  Many are not.

Reply:
Thanks. I have updated the code now.